api/model/src/test/java/org/projectnessie/model/TestReference.java

```diff
@@ -29,8 +35,13 @@ class TestReference {
     "a/b,11223344,a/b@11223344",
     "a/b,,a/b@",
     ",11223344,@11223344",
+    "test,~10,test~10",
+    "test,*123456,test*123456",
+    "test/path,~10,test/path@~10",
+    "test,deadbeef,test@deadbeef",
```

Comment:
Could you add a case for a `DETACHED` ref with timestamps and/or parents/merge parents?